docs/source/platform/schema-registry.md

```diff
@@ -36,9 +34,9 @@ npm install --global apollo
 
 Once the `apollo` command is installed, the `apollo service:push` command is used to publish a schema to Apollo Engine.
 
-To push a service, start the GraphQL server and run the following command, substituting the appropriate GraphQL endpoint URL and an API key:
+To push a service, start the GraphQL server and run the following command, substituting the appropriate GraphQL endpoint URL and API key:
 
-> An API key can be obtained from a service's _Settings_ menu within the [Engine dashboard](https://engine.apollographql.com/).
+> An API key can be obtained from a service's _Settings_ menu within the [Apollo UI](https://engine.apollographql.com/).
```

Comment:
Are we ready to start talking about the "Apollo UI" publicly? Should we do a whole migration of the world...?

Reply:
We aren't!